securebanking-openbanking-uk-rs-obie-api/src/main/java/com/forgerock/securebanking/openbanking/uk/rs/api/obie/payment/v3_0/domesticscheduledpayments/DomesticScheduledPaymentsApi.java

```diff
@@ -80,8 +80,11 @@
             @ApiParam(value = "A detached JWS signature of the body of the payload.", required = true)
             @RequestHeader(value = "x-jws-signature", required = true) String xJwsSignature,
 
+            @ApiParam(value = "The ID of the account that the payment is being made from.")
+            @RequestHeader(value = "x-ob-account-id", required = true) String xAccountId,
```

Comment:
Potentially controversial adding this custom header for only the scheduled payment & standing order endpoints, but without this I can't link the data to the relevant account.